coderd/database/models.go

```diff
@@ -35,6 +36,45 @@ func (e *APIKeyScope) Scan(src interface{}) error {
 	return nil
 }
 
+type NullAPIKeyScope struct {
```

Comment:
I don't see us ever using these Null forms of our enums. Is there a way to prevent their generation?

Reply:
Huh I can't find any option to toggle this off. I wonder if it's just a new sqlc thing.

Reply:
Ok I confirmed this is just new behavior.

https://github.com/kyleconroy/sqlc/pull/1485

Unfortunate we cannot pick and choose since we usually don't allow nulls. But :shrug:

Reply:
Oh yeah, I was a bit surprised by that behavior too. I'd have preferred to only generate the nullable one on-demand. Should be quite doable via a PR though, if it annoys us.